### DISCUSSION TITLE
Is there a rough estimate for when Functions release?

### DISCUSSION THREAD

**Discussion by erlendtryti** [supabase/supabase#1899] [Questions] [answered]
.

**Comment by inian** [comment, marked as answer]
Hi @erlendtryti 

Thanks for your interest in functions! We have many developers express interest in functions so we are working furiously to bring it to you all. With that being said, I can't quite give you an exact timeline. Roughly, we'll have something that early adopters can use in a few months.

For the latest on functions, you can leave your email here (https://supabasesubscribe.vercel.app/subscription/functions) and we'll keep you up to date!
